### DISCUSSION TITLE
How can commits show "github actions bot" instead of Unknown ?

### DISCUSSION THREAD

**Discussion by aahnik** [community/community#160496] [Actions] [answered]
![image](https://user-images.githubusercontent.com/66209958/115133630-6eeb1180-a027-11eb-9c3d-9b117f4094a5.png)

**Comment by peaceiris** [comment, marked as answer]
```sh
git config user.name 'github-actions[bot]'
git config user.email 'github-actions[bot]@users.noreply.github.com'
```

e.g. https://github.com/peaceiris/actions-gh-pages/blob/main/images/committer_github_actions_bot.jpg

**Reply by aahnik** [reply]
i have seen your action earlier. its awesome. planning to use it , instead of doing manually

**Comment by hoangtien2k3** [comment]
- ``` git config user.name 'github-actions[bot] ```
 - ```git config user.email 'github-actions[bot]@users.noreply.github.com'```

**Reply by qoomon** [reply]
I've created a handy action for that use case Feel free to have look at https://github.com/qoomon/actions--setup-git

**Comment by jxu** [comment]
Nowadays [actions/checkout recommends](https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token) 

          git config user.name "github-actions[bot]"
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"

**Comment by Bashirsaba** [comment]
git config user.name 'github-actions[bot]'
git config user.email 'github-actions[bot]@users.noreply.github.com'
